railties/lib/rails/commands/temp/temp_command.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require "rails/commands/temp/helpers"
+
+module Rails
+  module Command
+    class TempCommand < Base # :nodoc:
+      include Rails::Command::Temp::Helpers
+      namespace "tmp"
+      desc :clear, "Clear cache, socket and screenshot files from tmp/ (narrow w/ tmp:cache:clear, tmp:sockets:clear, tmp:screenshots:clear)"
+      def clear
+        Rails::Command.invoke "tmp:cache:clear"
+        Rails::Command.invoke "tmp:sockets:clear"
+        Rails::Command.invoke "tmp:screenshots:clear"
+        Rails::Command.invoke "tmp:storage:clear"
+      end
+
+      # desc :create, "Create tmp directories for cache, sockets, and pids"
```

Comment:
I think we can uncomment this.  It's not commented out for the current Rake task.

Reply:
Done.